addons/account/static/src/components/mail_attachments/mail_attachments.xml

```diff
@@ -17,11 +17,10 @@
                     'download': !file.placeholder ? '' : undefined,
                    }"
                 >
-                    <span class="o_image o_hover"
-                          style="height:15px;width:30px;"
-                          t-att-data-mimetype="file.mimetype"
-                          t-att-data-ext="ext"
-                          role="img"/>
+                    <img t-if="isImage(file)" t-att-src="'/web/image/' + file.id"
+                        class="o_preview_image o_hover object-fit-cover rounded"/>
+                    <span t-else="" t-att-data-mimetype="file.mimetype" t-att-data-ext="ext"
+                        class="o_image o_preview_image o_hover" role="img"/>
```

Comment:
Are all these image related classes and role still necessary? Since you will not have an image if it doesn't respect the condition.

Reply:
Hi again @thjo-odoo :slightly_smiling_face: 

The `t-att-data-mimetype` will actually set the mimetype image as background when the file isn't an image (cf `mimetypes.scss`). To make it work we need the `o_image` class.
![image](https://github.com/user-attachments/assets/a105ed82-3403-4f03-babf-fe238ed54f30)

I renamed the `o_image` to `o_preview_image` in the scss to avoid inheriting from all the `o_image` style from other files as we don't actually need a border or anything when we display the preview.
And `o_hover` to have the hover effect :slightly_smiling_face: 
So yes we still need everything :smile:

Reply:
@thjo-odoo Actually removing the `account` module part as, in this module, the preview has a specific width and height which is way too small to allow the image preview.
https://github.com/odoo/odoo/blob/master/addons/account/static/src/components/mail_attachments/mail_attachments.xml#L21
![image](https://github.com/user-attachments/assets/ea595559-8b15-44f0-9d9d-cb5a685b538f)